dashboards/system_tables/docs/hourly_utilization.md

```diff
@@ -45,3 +45,10 @@ This time series graph breaks down the overall slot utilization by user. This ca
 This chart displays the average usage for every hour, sorted by days with the most slot usage.
 
 ![By Top Usage](../images/hourly_utilization/by_top_usage.png)
+
+
+### Hourly Heatmap
+This chart displays the overall slot utilization broken down by day and hour.
+Cell color scales based on utilization and the largest utilization values are shown in black text.
```

Comment:
` largest utilization values are shown in black text` would be more clear if we said are shown in intense red boxes